sdk/monitor/Azure.Monitor.Ingestion/tests/Samples/LogDataAndQuery.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Monitor.Query;
+using Azure.Monitor.Query.Models;
+
+namespace Azure.Monitor.Ingestion.Tests.Samples
+{
+    public partial class IngestionSamples: SamplesBase<IngestionClientTestEnvironment>
+    {
+        public void SetUpClient()
+        {
+            #region Snippet:CreateLogsIngestionClient
+            Uri dataCollectionEndpoint = new Uri("...");
+            TokenCredential credential = new DefaultAzureCredential();
```

Comment:
Let's go for consistency in this snippet. Align on `var`.
```suggestion
            var dataCollectionEndpoint = new Uri("...");
            var credential = new DefaultAzureCredential();
```

Reply:
I thought for snippets we want as much detailed variable declaration as possible - so I made all the var's their explicit type. Maybe this is the wrong thing to do?

Reply:
Cases like this are an exception. Prefer `var` when instantiating, since the type is obvious.